api/controllers/MachineController.js

```diff
@@ -51,10 +59,18 @@ module.exports = {
    * @method users
    */
   users(req, res) {
-    Machine.find()
-      .then((machines) => {
-        return res.ok(machines);
-      })
-      .catch((err) => res.negotiate(err));
+    if (req.user.isAdmin) {
+      Machine.find()
```

Comment:
Actually, `/api/machines/users` MUST always return `getMachineForUser` regardless of user's status.